gokart/slack/event_aggregator.py

```diff
@@ -1,44 +1,49 @@
 import os
-from collections import defaultdict
 from logging import getLogger
+from typing import List, TypedDict
 
 import luigi
 
 logger = getLogger(__name__)
 
 
+class FailureEvent(TypedDict):
+    task: str
+    exception: str
+
+
 class EventAggregator(object):
 
     def __init__(self) -> None:
-        self._event_queue = defaultdict(list)
+        self._success_events: List[str] = []
+        self._failure_events: List[FailureEvent] = []
```

Comment:
Previous `_event_queue` has different types by the keys. It is difficult to specify the types. So, I separate the lists.